crates/wasm-mutate-stats/src/bin/wasm-mutate-stats.rs

```diff
@@ -0,0 +1,327 @@
+                let mut strjson = String::new();
+                file.read_to_string(&mut strjson)?;
+
+                let deserialized: Report = serde_json::from_str(&strjson)?;
+                Ok(deserialized)
+            })
+            .or_else(|_| {
+                let created = File::create(format!("{}.report.json", name));
+                created.and_then(|mut f| {
+                    let newreport = Report {
+                        input: name.to_string(),
+                        wasm,
+                        rotations: 0,
+                        seed: 0,
+                        hashes: HashMap::new(),
+                        configs: self
+                            .compiler_configs
+                            .iter()
+                            .map(|c| {
+                                (
+                                    c.join(""),
+                                    ConfigReport {
+                                        hashes: HashMap::new(),
+                                    },
+                                )
+                            })
+                            .collect(),
+                        errors: Vec::new(),
+                    };
+                    let serialized = serde_json::to_string(&newreport).unwrap();
+                    f.write(serialized.as_bytes())?;
+                    Ok(newreport)
+                })
+            })
+    }
+
+    fn save_report(&self, report: &Report, name: &String) {
+        let created = File::create(format!("{}.report.json", name));
+        created
+            .and_then(|mut f| {
+                let serialized = serde_json::to_string(report).unwrap();
+                f.write(serialized.as_bytes())
+            })
+            .expect("Report could not be written");
+    }
+
+    fn run_worker(&self, start: usize, end: usize) {
+        let mut wasmmutate = WasmMutate::default();
+        let mut corpus = self.corpus[start..end].to_vec();
+
+        loop {
+            match corpus.pop() {
+                Some((name, data)) => {
+                    println!("Wasm input {}", name);
+                    let mut wasm = data;
+
+                    let mut report = self.load_report(&name, wasm.clone()).unwrap();
+
+                    wasmmutate.seed(report.seed);
+                    let mut rng = SmallRng::seed_from_u64(report.seed);
+                    // mutate the initial wasm as many rotations are in the loaded report
+                    print!("Reaching previous state...");
+                    for _ in 0..report.rotations {
+                        match wasmmutate.run_with_rng(&wasm, &mut rng) {
+                            Ok(mutated) => {
+                                wasm = mutated;
+                            }
+                            Err(_) => { /* bypass */ }
+                        }
+                    }
+                    println!("Reached");
+                    while report.rotations <= ROTATIONS {
```

Comment:
Supporting time limits, instead of only count limits, would allow us to easily make statements like "`wasm-mutate` generates X unique Wasm modules per hour"

Also, the count should be configurable as well (or missing if we are doing a time limit, and vice versa).